### PR TITLE
Remove invalid step from README.md

### PR DESCRIPTION
**Pull request by anupama-pathirage**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,7 +220,6 @@ Change the value of the `FRONTEND_SVC_URL` variable in `ui/src/lib/api.js` to th
 * Set up the email service with email credentials as explained above.
 * Build and publish the `client_stubs` and `money` modules to the local central as follows.
 ```bash
-cd module
```
Comment (mohanvive) [inline README.md:223]: I believe this step is written assuming user will be at the root folder. And the user has to go inside each module `client_stubs` and `money` , build the jars and push to local repository. @xlight05 @dilanSachi Is that correct?

**Review by mohanvive — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dilanSachi) [reply README.md:223]: yes. Added `module` as a placeholder for `client_stubs` and `money`

Comment (anupama-pathirage) [reply README.md:223]: I see, Then we better rename. This was a comment from a user who tried to built it and said no such folder named `module` within `money`

**Review by dilanSachi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (anupama-pathirage) [reply README.md:223]: Added a sentence. Pls check @mohanvive / @dilanSachi

**Pull request merged**